book/10-git-internals/sections/objects.asc

```diff
@@ -149,9 +149,9 @@ $ git cat-file -p 99f1a6d12cb4b6f19c8655fca46c3ecf317074e0
 ====
 Depending on what shell you use, you may encounter errors when using the `master^{tree}` syntax.
 
-In PowerShell or CMD on Windows, the `^` character is used for escaping, so you have to double it to avoid this: `git cat-file -p master^^{tree}`.
+In CMD on Windows, the `^` character is used for escaping, so you have to double it to avoid this: `git cat-file -p master^^{tree}`. When using PowerShell parameters using {} characters have to be quoted to avoid the parameter being parsed incorrectly: `git cat-file -p 'master^{tree}'`.
```

Comment:
Perfect. One small request: can you reformat this so that each sentence is on its own line? That helps immensely with PR diffs in the future.